sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -198,47 +233,104 @@ public partial class RegistryArtifact
     {
         protected RegistryArtifact() { }
         public virtual string FullyQualifiedReference { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual System.Uri RegistryEndpoint { get { throw null; } }
         public virtual string RepositoryName { get { throw null; } }
+        public virtual Azure.Response Delete(string name, string reference, Azure.RequestContext context = null) { throw null; }
```

Comment:
Remove name.  Remove reference?  Make RequestContext required if we have an ambiguous call site.